src/shapes/SurfaceCircle.js

```diff
@@ -162,6 +162,19 @@ define(['../error/ArgumentError',
             }
         };
 
+        // Internal use only. Intentionally not documented.
+        SurfaceCircle.prototype.getReferencePosition = function () {
+            return this.center;
+        };
+
+        // Internal use only. Intentionally not documented.
+        SurfaceCircle.prototype.moveTo = function (globe, position) {
+            var locations = [];
+            locations.push(new Location(this.center.latitude, this.center.longitude));
```

Comment:
Could the center property be used directly instead of creating a new location object?